vmss.tf

```diff
@@ -110,6 +110,11 @@ resource "azurerm_linux_virtual_machine_scale_set" "vmss" {
     type = "SystemAssigned"
   }
 
+  automatic_instance_repair {
+    enabled      = true
+    grace_period = "PT20M"
```

Comment:
why? we dont want azure to replace instances for us
we detect that something wrong and properly drain, azure can make bad decision and kill the cluster